msm/skill_entry.py

```diff
@@ -130,9 +130,9 @@ def __init__(self, name, path, url='', sha='', msm=None):
             self.name = self.meta_info['name']
         else:
             self.name = basename(path)
-
-        self.author = self.extract_author(url) if url else ''
-        self.id = self.extract_repo_id(url) if url else self.name
+        from_github = (url.split("/")[-3] == 'github.com') if url else False
+        self.author = self.extract_author(url) if from_github else ''
```

Comment:
Wonder if we should make this `else 'unknown'`

I believe it's used when creating the repo directory (`id.author`)

Reply:
Unknown may make more sense, I just kept it as an empty string to match what was there before.

Reply:
Ah I see. Maybe keep it as it is for now then.